spec/link-spec.coffee

```diff
@@ -25,9 +25,8 @@ describe 'apm link/unlink', ->
       runs ->
         expect(fs.existsSync(path.join(atomHome, 'packages', path.basename(packageToLink)))).toBeTruthy()
         expect(fs.realpathSync(path.join(atomHome, 'packages', path.basename(packageToLink)))).toBe fs.realpathSync(packageToLink)
-        callback.reset()
 
-      runs ->
+        callback.reset()
```

Comment:
I combined the consecutive `runs` blocks in this spec file for clarity. Usually we don't have multiple `runs` block after each other since it essentially the same as having it all in a single `runs` block.